test/test_main.py

```diff
@@ -212,7 +212,6 @@ def _assess_setup_hooks(event_name):
             self.assertTrue(event_name in files)
 
             for event_hook in event_hooks:
```

Comment:
I think we also need to make sure that there are no symlinks created for non-juju events here.